tests/network/primary_pod_network.go

```diff
@@ -167,7 +172,13 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 				var vmi *v1.VirtualMachineInstance
 
 				BeforeEach(func() {
-					vmi = setupVMI(virtClient, vmiWithMasqueradeBinding())
+					vmi = setupVMI(
+						virtClient,
+						libvmi.NewAlpine(
+							libvmi.WithNetwork(v1.DefaultPodNetwork()),
+							libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
```

Comment:
I know that masquerade is only for primary interfaces but this way it has the same interface as the BridgeBinding
```suggestion
						libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding(libvmi.DefaultInterfaceName)),
```

Another option is to have a `DefaultInterfaceDeviceWithMasqueradeBinding` and `DefaultInterfaceDeviceWithBridgeBinding`

Reply:
Done